tuf/data/timestamp.go

```diff
@@ -64,9 +61,11 @@ func NewTimestamp(snapshot *Signed) (*SignedTimestamp, error) {
 	return &SignedTimestamp{
 		Signatures: make([]Signature, 0),
 		Signed: Timestamp{
-			Type:    TUFTypes["timestamp"],
-			Version: 0,
-			Expires: DefaultExpires("timestamp"),
+			SignedCommon: SignedCommon{
+				Type:    TUFTypes["timestamp"],
```

Comment:
nit: can we use the `CanonicalTimestampRole` constant here and below for `Expires:`?